flink-clients/src/main/java/org/apache/flink/client/program/ContextEnvironment.java

```diff
@@ -74,7 +74,6 @@ public String getExecutionPlan() throws Exception {
 
 	@Override
 	public void startNewSession() throws Exception {
-		client.endSession(jobID);
```

Comment:
(2/2) Remove `endSession(JobID)` use point. Please check that this doesn't change current behaviors.